### PR TITLE
Tree generation

### PR DESCRIPTION
**Pull request by windy1**
Added some basic tree generation (Squashed and signed commit's @Afforess )

Signed-off-by: WalkerCrouse wjcrouse913@aim.com

### CODE REVIEW

File: src/main/java/org/spout/vanilla/generator/normal/TreePopulator.java [hunk 1]
```diff
@@ -25,26 +25,220 @@
...
+    public void generateNormalTree(Chunk c, Point point) {
+    	int x = (int) point.getX();
+    	int y = (int) point.getY();
+    	int z = (int) point.getZ();
+    	short wood = VanillaBlocks.log.getId();
+    	short leaves = VanillaBlocks.leaves.getId();
+    	World world = c.getWorld();
+    	
+    	// Trunk
+    	c.setBlockId(x, y+1, z, wood, world);
```
Comment (Top-Cat) [inline src/main/java/org/spout/vanilla/generator/normal/TreePopulator.java]: I maintain looping here would be better...

File: src/main/java/org/spout/vanilla/generator/normal/TreePopulator.java [hunk 2]
```diff
@@ -25,26 +25,220 @@
...
+    	c.setBlockId(x-1, y+5, z-1, leaves, world);
+    	c.setBlockId(x-1, y+5, z+1, leaves, world);
+    	c.setBlockId(x+2, y+5, z, leaves, world);
+    	c.setBlockId(x-2, y+5, z, leaves, world);
+    	c.setBlockId(x, y+5, z+2, leaves, world);
+    	c.setBlockId(x, y+5, z-2, leaves, world);
+    	c.setBlockId(x+2, y+5, z+2, leaves, world);
+    	c.setBlockId(x+2, y+5, z-2, leaves, world);
+    	c.setBlockId(x+2, y+5, z-2, leaves, world);
+    	c.setBlockId(x+2, y+5, z+2, leaves, world);
```
Comment (Top-Cat) [inline src/main/java/org/spout/vanilla/generator/normal/TreePopulator.java]: This looks remarkably similar to line 85

File: src/main/java/org/spout/vanilla/generator/normal/TreePopulator.java [hunk 3]
```diff
@@ -25,26 +25,220 @@
...
+    	c.setBlockId(x+1, y+6, z, leaves, world);
+    	c.setBlockId(x-1, y+6, z, leaves, world);
+    	c.setBlockId(x, y+6, z+1, leaves, world);
+    	c.setBlockId(x, y+6, z-1, leaves, world);
+    	c.setBlockId(x+1, y+6, z+1, leaves, world);
+    	c.setBlockId(x+1, y+6, z-1, leaves, world);
+    	c.setBlockId(x-1, y+6, z-1, leaves, world);
+    	c.setBlockId(x-1, y+6, z+1, leaves, world);
+    	
+    	// Third leaves layer
```
Comment (Top-Cat) [inline src/main/java/org/spout/vanilla/generator/normal/TreePopulator.java]: Third and fourth are just a y offset, loop time?

**Comment by windy1** [comment]
Stand bye while I fail at git...

**Comment by windy1** [comment]
Alright, all done @Top-Cat

**Comment by Top-Cat** [comment]
The file is still tabs and spaces from what I can see...

**Comment by windy1** [comment]
@Top-Cat Okay, for real this time XD

**Comment by Top-Cat** [comment]
**Top_Cat** _giggles_

**Pull request merged**